lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -115,44 +150,55 @@ def check_ndjson(ndjson, type)
         assert_response_unauthorized reply
       end
 
-      test 'Server shall return "202 Accepted" and "Content-location" for $export operation' do
+      test 'Server shall return "202 Accepted" and "Content-location" for $export operation with parameters' do
         metadata do
           id '02'
+          link 'https://build.fhir.org/ig/HL7/bulk-data/export/index.html#query-parameters'
+          desc %(
+          )
+        end
+
+        check_export_kick_off('Patient', search_params: { '_type' => 'Patient' })
+      end
+
+      test 'Server shall return "202 Accepted" and "Content-location" for $export operation' do
+        metadata do
+          id '03'
           link 'https://build.fhir.org/ig/HL7/bulk-data/export/index.html#bulk-data-kick-off-request'
           desc %(
           )
         end
 
+        skip 'Skip testing $export without parameters' if @server_support_type_parameter && !@run_all_kick_off_tests
```

Comment:
```suggestion
        skip 'Skip testing $export without parameters' if @server_support_type_parameter && !run_all_kick_off_tests
```
Now that you have an `attr_accessor`, use it here. This protects against typos in instance variable names. If someone typed `@run_all_kickoff_tests` instead of `@run_all_kick_off_tests`, they'd get `nil` and the tests would keep running and they may or may not notice that this test is no longer working as intended. A typo in the name of an accessor, however, would raise an exception so that the error could be discovered immediately.